galaxyui/src/app/my-imports/import-detail/import-detail.component.ts

```diff
@@ -32,13 +34,26 @@ export class ImportDetailComponent implements OnInit, AfterViewInit {
 
     private _importTask: Import;
     private _refreshing: boolean;
+    private canImport: boolean;
 
     scroll = false;
 
     ImportState: typeof ImportState = ImportState;
 
     @Input()
     set importTask(data: Import) {
+        this.canImport = false;
+
+        if (data) {
+            this.authService.me().subscribe(
+                (me) => {
+                    console.log(me.username);
+                    console.log(data.summary_fields.owner.username);
```

Comment:
Remove console.log() where not needed. Presumably these were used to peek at a data structure during development.

Reply:
oops